janusgraph-hadoop-parent/janusgraph-hadoop-core/src/test/java/org/janusgraph/hadoop/AbstractInputFormatIT.java

```diff
@@ -77,11 +78,10 @@ public void testReadWideVertexWithManyProperties() throws Exception {
         GraphTraversalSource t = g.traversal(GraphTraversalSource.computer(SparkGraphComputer.class));
         assertEquals(numV, (long) t.V().count().next());
         propertiesOnVertex = t.V().valueMap().next();
-        valuesOnP = (List)propertiesOnVertex.values().iterator().next();
-        assertEquals(numProps, valuesOnP.size());
-        for (int i = 0; i < numProps; i++) {
-            assertEquals(Integer.toString(i), valuesOnP.get(i).toString());
-        }
+        final Set<?> observedValuesOnP = ImmutableSet.copyOf((List)propertiesOnVertex.values().iterator().next());
+        assertEquals(numProps, observedValuesOnP.size());
+        // order may not be preserved in multi-value properties
+        assertEquals(0, Sets.symmetricDifference(observedValuesOnP, ImmutableSet.copyOf(valuesOnP)).size());
```

Comment:
What about `assertTrue(Sets.symmetricDifference(...).isEmpty());` ?

Or `assertThat(observedValuesOnP).equals(ImmutableSet.copyOf(valuesOnP))` (or similar)? This appears to be testing for set equality; do we need a symmetric difference if we only care that it's 0?